src/common.js

```diff
@@ -10,6 +10,9 @@ export const IMPERSONATION_REAL_USER_AUTH_KEY = '_stitch_impers_real_ua';
 export const USER_AUTH_COOKIE_NAME = 'stitch_ua';
 export const DEFAULT_STITCH_SERVER_URL = 'https://stitch.mongodb.com';
 
+// VERSION is substituted with the package.json version number at build time
+export const SDK_VERSION = VERSION;
```

Comment:
does `VERSION` get a default value from somewhere, or can we give it one (like "unknown" or something)? as-is it looks like if you were to try to import the module directly from `src`, JS might complain that `VERSION` was never declared, i think.